### ISSUE TITLE
Deprecated capability flag canSignalEvents of FMU CS 1.0.1

### ISSUE THREAD

**Issue by beutlich** [modelica-tools/FMUComplianceChecker#38]
The FMU Compliance Checker v2.0.4 raises an XML error if an FMU CS 1.0.x mentions the canSignalEvents capability flag in modelDescription.xml.

```
[ERROR][FMI1XML] Detected on line:555 of modelDescription.xml
[ERROR][FMI1XML] Found capability flag canSignalEvents which have been deprecated as it fills no function 
```

Since this flag was only deprecated for FMI 1.0.1 (see https://github.com/modelica/fmi-standard/issues/370) and a valid FMU 1.0.0 also is a valid FMU 1.0.1 this should not be considered as error. Rather I propose to consider it as XML warning.

@TorstenBlochwitz FYI
